Task10_Auth/src/main/java/UsersClient.java

```diff
@@ -43,9 +45,25 @@ public static ResponseWrapper createUser(UsersDto usersDto) {
         return responseWrapper;
     }
 
-    public static ResponseWrapper getUsers() {
-        HttpGet httpGet = new HttpGet(PropertiesReader.get("appURI") + PropertiesReader.get("usersURI"));
-        httpGet.addHeader("Authorization", AuthSingleton.getInstance().getReadToken());
+    public static ResponseWrapper getUsers(Integer olderThan, Integer youngerThan, UsersDto.Sex sex) {
```

Comment:
I think you can avoid a lot of calls of:
`UsersClient.getUsers(null, null, null)`
in tests if you will add in client's code one more method:
`public static ResponseWrapper getUsers() {
return getUsers(null, null, null);
}`
Also, such approach grant that in future you will not need to change many places with the same code

Reply:
done